pom.xml

```diff
@@ -156,6 +150,12 @@
             <version>${java.tuples.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>${org.json.version}</version>
```

Comment:
As this lib might be used by spring boot app, and spring boot dependencies manage `jackson`, wonder how about migrate to Jackson in the future?